docs/source/array-gufunc.rst

```diff
@@ -33,8 +30,18 @@ NumPy generalized ufunc
     w, v = np.linalg._umath_linalg.eig(x, output_dtypes=(float, float))
 
 
-Wrap own Python function
-~~~~~~~~~~~~~~~~~~~~~~~~
+Create Generalized UFuncs
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+As of NumPy 1.17 it is difficult to create your own GUFuncs without going into
```

Comment:
Was there a specific change in NumPy 1.17, or is this more along the lines of "With recent versions of NumPy it is difficult to create..."?